src/core/state/currentSettings/featureFlags.ts

```diff
@@ -22,6 +23,7 @@ export const featureFlagsStore = createStore<FeatureFlagsStore>(
       hw_wallets_enabled: true,
       command_k_internal_shortcuts_enabled: false,
       custom_rpc: false,
+      nfts_enabled: false,
```

Comment:
you need to bump the persist version to make this new feature flag available 
i spent some time trying to make it appear